beamer/agent/config.py

```diff
@@ -110,9 +113,15 @@ def load(config_path: Path, options: dict[str, Any]) -> Config:
         raise ConfigError(f"missing settings: {missing}")
 
     rpc_urls = {}
+    poll_period_per_chain = {}
 
     for chain_name, chain_info in config["chains"].items():
-        rpc_urls[chain_name] = URL(chain_info["rpc-url"])
+        rpc_url = chain_info.get("rpc-url")
+        if rpc_url is not None:
+            rpc_urls[chain_name] = URL(rpc_url)
```

Comment:
I would omit the check for existence of `rpc_url` here, let us make it crash here, rather than later in the process, when the agent tries to start.

This will anyway be addressed later by proper config validity checks.